poglink/cogs/rates.py

```diff
@@ -74,43 +56,20 @@ async def get_current_rates(url):
             except Exception as e:
                 raise RatesFetchError(e) from e
 
-    async def send_embed(self, description, url, title=None):
+    async def send_embed(self, description, rates_dict, **kwargs):
+        url = rates_dict.get("url", DEFAULT_URL)
+        server_name = rates_dict.get("server_name", DEFAULT_SERVER_NAME)
+        server_color = rates_dict.get("color", DEFAULT_COLOR)
```

Comment:
I added defaults to some of these because I didn't actually update with concrete values in all the config fixtures and things were acting funny when they were `None`. Might want some more friendly error messages thrown here instead when important params are missing, but I'll leave that to you to decide.